src/app/components/listing/details/criteria/criterion.jsx

```diff
@@ -207,10 +208,10 @@ function ChplCriterion(props) {
                       Retired |
                     </>
                   )}
-                  <div className={classes.criterionNumber}>
-                    {criterion.criterion.number}
-                  </div>
                 </Typography>
+                <div className={classes.criterionNumber}>
```

Comment:
Should this be a `<Typography>` tag instead of a div? Both are block elements, but making it a typography would be more correct in a semantic sense